data/hai/hai outfits.txt

```diff
@@ -11,6 +11,40 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Phantom Pallet"
+	plural "Phantom Pallets"
+	category "Systems"
+	cost 12000
+	thumbnail "outfit/phantom pallet"
+	"mass" 0.2
+	"unplunderable" 1
+	"energy consumption" 0.016667
+	"heat generation" 0.03334
+
+	# Replacing 30% of your cargo bay with these triples your cargo scan time.
```

Comment:
Are you sure about this? It looks to me like replacing 30% of your cargo bay with these would slightly more than quintuple your cargo scan time.

Suppose you have X cargo.
If you replace 30% of your cargo with these, you now have 0.3X of them.
That's a cargo scan opacity of (0.3X * 15 = 4.5X).
So you now have 0.7X cargo and 4.5X cargo scan opacity, which means in total your effective cargo hold for scanning should be 5.2X the size.

Reply:
Look at the scan formula. The scanning rate is not linearly dependent on the depth. It is proportional to the depth raised to the power of -0.666666666666666666666.

Reply:
Oh, right. The comment is correct, then (5.2 ^ 0.666... = 3.001).